api/pyproject.toml

```diff
@@ -70,12 +70,12 @@ djangorestframework-recursive = "~0.1.2"
 packaging = "~23.0"
 chargebee = "~2.7.7"
 python-http-client = "~3.3.7"
-django-health-check = "~3.14.3"
-django-admin-sso = "~3.1.0"
+django-health-check = "~3.17.0"
+django-admin-sso = "~5.0"
```

Comment:
We should make sure to manually test this

Reply:
Noted! Once we figure out how to use the new trench classes, we can do that.